src/games/strategy/triplea/ResourceLoader.java

```diff
@@ -98,14 +99,16 @@ public boolean match(final File f) {
         return f.exists();
       }
     });
-    if (existing.size() > 1) {
-      System.out.println("INFO: Found too many files for: " + mapName + "  found: " + existing);
-    }
+
+
     // At least one must exist
     if (existing.isEmpty()) {
       return Collections.emptyList();
     }
     final File match = existing.iterator().next();
+    if (existing.size() > 1) {
```

Comment:
You're right, #620 would have fixed.  We also have a second problem, if the "map" folder exists, so will the parent. So in the git clone case, we will always get this warning.